config/application.rb

```diff
@@ -63,5 +63,7 @@ class Application < Rails::Application
     }
 
     config.active_record.yaml_column_permitted_classes = [BigDecimal, Date, Symbol]
+
+    config.email_regexp = /\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+\z/
```

Comment:
Might be worth matching the regex used on the Notifications API: https://github.com/alphagov/notifications-utils/blob/ea93a8989028de2529f651a696bab13f9599a44c/notifications_utils/__init__.py#L14

Reply:
If it were Python but since I'd have to modify it, I'd rather keep this more explicit version.